schemas/polygonValuesAxis.json

```diff
@@ -28,7 +28,7 @@
                         "minItems" : 1
                     }
                 },
-                "coordinates": { }
+                "coordinates": { "const" : ["x", "y"] }
```

Comment:
The CovJSON spec only says that the "axis value MUST be a GeoJSON Polygon coordinate array", and in GeoJSON it defines the base position as "A position is an array of numbers.  There MUST be two or more elements."
So I think semantically the `["x","y"]` constraint still belongs to the common domain types, not the core. There may be a new domain type in the future that extends this to more coordinates.

Reply:
Yes, good point, I agree - I'll change that